opentelekomcloud/data_source_opentelekomcloud_kms_data_key_v1.go

```diff
@@ -0,0 +1,70 @@
+package opentelekomcloud
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/huaweicloud/golangsdk/openstack/kms/v1/keys"
+	"time"
+)
+
+func dataSourceKmsDataKeyV1() *schema.Resource {
+	return &schema.Resource{
+		Read: resourceKmsDataKeyV1Read,
+
+		Schema: map[string]*schema.Schema{
+			"key_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"encryption_context": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"datakey_length": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"plain_text": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
If this isn't input, it doesn't need to be set `Optional` here.

Reply:
Done